.eslintrc.js

```diff
@@ -1,38 +1,3 @@
 module.exports = {
-    "env": {
-        "es6": true,
-        "node": true
-    },
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "project": "tsconfig.json",
-        "sourceType": "module"
-    },
-    "plugins": [
-        "@typescript-eslint",
-        "import"
-    ],
-    "extends": [
-        "eslint:recommended",
-        "plugin:@typescript-eslint/recommended",
-        "plugin:@typescript-eslint/recommended-requiring-type-checking"
-    ],
-    "rules": {
-        "no-unused-vars": "off",
-        "no-useless-escape": "off",
-        "no-inner-declarations": "off",
-        "no-case-declarations": "off",
-        "@typescript-eslint/prefer-regexp-exec": "off",
-        "@typescript-eslint/no-unused-vars": [
-            "error",
-            { "argsIgnorePattern": "^_" }
-        ],
-        "@typescript-eslint/no-inferrable-types": "off",
-        "@typescript-eslint/unbound-method": "off",
-        "@typescript-eslint/no-unnecessary-type-assertion": "off",
-        "@typescript-eslint/restrict-template-expressions": "off",
-        "@typescript-eslint/no-namespace": "off",
-        "@typescript-eslint/ban-types": "off",
-        "@typescript-eslint/require-await": "off"
-    }
+    "extends": "@microsoft/eslint-config-azuretools"
```

Comment:
Since it looks like all the `import/no-internal-modules` issues are dealing with the `antlr4ts` package, I think we should override the rule like so: 

```suggestion
    "extends": "@microsoft/eslint-config-azuretools"
    "rules": {
        "import/no-internal-modules": [ "error", {
            "allow": [ "antlr4ts/**" ]
        } ]
    }
```